ingestion/functions/parsing/japan/input_event.json

```diff
@@ -0,0 +1,5 @@
+{
+    "s3Bucket": "epid-sources-raw",
+    "s3Key": "japan/content.json",
```

Comment:
this is where your error comes from, the source id gets extracted from the s3 key, you need a `<source uuid>/year/month/date/content.json` format here

Reply:
OK thank you for looking into this. What exactly constitutes the source uuid? Is this just 'japan'? and the 'yeah/month/date/' is that today's date? I am slightly confused by this as in the example parser it seems 'japan-test-id/content.json' is enough as an s3 key - @axmb could you perhaps shed some light on this?

Reply:
This https://github.com/globaldothealth/list/pull/747 should make things simpler, the source UUID is the "ID" column in the sources table (url: /sources in the UI).
The first part of the s3 path/key should be a uuid because the script gets it from there right now, this is not really needed and instead we should provide that in the input event of the lambda, this is what the PR above does. You can look at the india or zurich parser's input_event.json in the same directory as your japan one to see a correct s3key in the meantime.

Reply:
You are going to have to clarify that - what exactly do you mean by 'url: /sources in the UI'? Sorry, I feel like an idiot with all these questions, still getting my head around everything

Reply:
sorry if it was confusing, you don't have to care about the UI bit, I just merged the PR I mentioned, please merge and check out the other parsers examples, it should make it clear what's a valid uuid.

Reply:
Sure - but where do I actually get the source UUID from?

Reply:
You can make it up in your input_event.json, it doesn't have to be a real UUID of a real source when doing so anymore now that I merged that other PR.
Probably the docs could mention that as it is confusing. I can try to add some around how to test the retrieval lambda that also needs a source uuid next week if @axmb doesn't get to it before I do.

Reply:
Ah OK thank you!

Reply:
Apologies Anya, we'll clarify this bit. ITMT, let us know if Tim's change doesn't unblock you.

Reply:
Thanks a lot for your help guys!